Sources/AvatarView.swift

```diff
@@ -48,7 +48,7 @@ open class AvatarView: UIView {
         let context = UIGraphicsGetCurrentContext()!
         
         //// Text Drawing
-        let textRect = CGRect(x: 5, y: 6, width: 20, height: 20)
+        let textRect = CGRect(x: 0, y: 0, width: 30, height: 30)
```

Comment:
I like this idea but let's go ahead an make some improvements here. Since we are really just using the views frame instead of hardcoding this size. So remove the `textRect` and replace any reference to it with the  `view.frame`.

Reply:
This way if a different size Avatar is used it will adjust accordingly. There is still no safety in preventing a large `font size` but at least it is an improvement.